### ISSUE TITLE
Sharing multiple objects

### ISSUE THREAD

**Issue by omerbaskurt** [mentrena/SyncKit#74]
Hi @mentrena , first of all, thank you for your efforts, it's been a great helping using this library.
I was trying to share multiple objects (company) in the example project and it stucks on uploading part. Is there any way to share same type of objects together, without having a parent class?

**Comment by mentrena** [comment]
Hey @omerbaskurt , just saw this. Was the issue fixed for you?

**Comment by omerbaskurt** [comment]
Yes, I'm sharing the parent class which has an array of objects and it works, thanks.
